packages/transactions/src/callHelpersContextParametrized.ts

```diff
@@ -131,3 +131,28 @@ export function createSendWithGasConstraints<A extends TxMeta, CC extends Contex
     );
   };
 }
+export function createSendWithGasConstraints1559<A extends TxMeta, CC extends ContextConnected>(
+  send: SendFunction<A>,
+  context: CC,
+  gasPrice$: GasPrice$,
+) {
+  return <B extends A>(callData: TransactionDef<B, CC>, args: B): Observable<TxState<B>> => {
+    return combineLatest(estimateGas(context, callData, args), gasPrice$).pipe(
+      first(),
+      switchMap(([gas, gasPrice]) => {
+        return createSendTransaction(send, context)(
+          {
+            ...callData,
+            options: (args1: B) => ({
+              ...(callData.options ? callData.options(args1) : {}),
+              gas,
+              maxPriorityFeePerGas: gasPrice,
+              maxFeePerGas: gasPrice,
```

Comment:
Aren't we supposed to have these like different values ?  OR at least configurable .. I'm not the EIP-1559 expert though

Reply:
In future they will be, but that is not a scope of this story
https://app.shortcut.com/oazo-apps/story/2733/change-transaction-format-in-oasis-borrow
in future gasPrice$ will return two values